common/lib/clear-cache-reference-data.test.js

```diff
@@ -0,0 +1,128 @@
+const proxyquire = require('proxyquire').noPreserveCache()
+
+const API_VERSION = require('../../config').API.VERSION
```

Comment:
It might be better to mock this within the `proxyquire` call too.

Reply:
I did consider it, done